src/swift/Time.swift

```diff
@@ -119,6 +119,40 @@ public func ==(a: DispatchWallTime, b: DispatchWallTime) -> Bool {
 	return a.rawValue == b.rawValue
 }
 
+// Returns m1 * m2, clamped to the range [Int64.min, Int64.max].
+// Because of the way this function is used, we can always assume
+// that m2 > 0.
+private func clampedInt64Product(_ m1: Int64, _ m2: Int64) -> Int64 {
+	assert(m2 > 0, "multiplier must be positive")
```

Comment:
`assert` is a noop in release builds. Should perhaps use `_precondition`?

Reply:
Not having the check in release builds is exactly what I want. The only calls to this method are internal to this file and the intent is only to document the assumption and to catch (admittedly unlikely) future changes that violate it.